crates/bitwarden-vault/src/cipher/linked_id.rs

```diff
@@ -16,22 +16,12 @@ pub enum LinkedIdType {
 }
 
 #[cfg(feature = "uniffi")]
-use crate::UniffiCustomTypeConverter;
-#[cfg(feature = "uniffi")]
-uniffi::custom_type!(LinkedIdType, u32);
-#[cfg(feature = "uniffi")]
-impl UniffiCustomTypeConverter for LinkedIdType {
-    type Builtin = u32;
-
-    fn into_custom(val: Self::Builtin) -> uniffi::Result<Self> {
-        let val = serde_json::Value::Number(val.into());
-        Ok(serde_json::from_value(val)?)
-    }
-
-    fn from_custom(obj: Self) -> Self::Builtin {
-        obj.into()
-    }
-}
+uniffi::custom_type!(LinkedIdType, u32, {
+    try_lift: |val| {
+        LinkedIdType::try_from(val).map_err(|e| e.into())
+    },
+    lower: |obj| obj.into(),
```

Comment:
suggestion: here too, I think we could skip `lower`

```suggestion
```

It doesn't cause any compilation errors and from what I can see in the source this should be supported

https://github.com/mozilla/uniffi-rs/blob/4e0a9e96a546014af4cecba1ca382c4074cb7fb2/uniffi_macros/src/custom.rs#L184-L197

Reply:
Done, also skipped the try_lift, as it's doing `Ok(try_into()?)`, which is basically the same thing I was doing mapping the error.

Reply:
Oh yeah true, nice!